bpf/include/bpf/config/node.h

```diff
@@ -16,3 +16,4 @@
 #include <node_config.h>
 
 NODE_CONFIG(union v6addr, router_ipv6, "Internal IPv6 router address assigned to the cilium_host interface")
+NODE_CONFIG(union v6addr, ipv6_loopback, "IPv6 loopback address")
```

Comment:
It looks like we ended up settling on `service_loopback_ipv4` in https://github.com/cilium/cilium/pull/38818. Rename this and follow the [same patterns](https://github.com/cilium/cilium/pull/38818/commits/3d057129899afe3b6e0e75743116f0cd4c0183da).